SPECS/liblogging/liblogging.spec

```diff
@@ -0,0 +1,78 @@
+Summary:        Logging Libraries
+Name:           liblogging
+Version:        1.0.6
+Release:        3%{?dist}
```

Comment:
We should also take #303, to ensure SRPMs build.

Reply:
Thanks. Agree we should pull this change, but will prefer to keep it separate since this merge is already large.